src/style.css

```diff
@@ -0,0 +1,76 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir-Book";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+  font-style: normal;
+}
+
+html {
+  font-family: "Avenir-Book", "Avenir-Heavy", sans-serif;
+}
+
+.search-bar-desktop {
+  position: relative;
+  margin: 20px 0;
+}
+
+.search {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: "Avenir-Heavy", sans-serif;
+}
+
+.icon {
+  position: absolute;
+  left: 26px;
+  top: 25px;
+}
+
+.bigSearch {
```

Comment:
You can remove this class and move all rules to .search selector. As a result inputs will have classes like this:
Big: only .search
Small: .search .smallSearch